packages/textfields/src/views/__snapshots__/Input.spec.js.snap

```diff
@@ -2,6 +2,7 @@
 
 exports[`Input renders RTL styling 1`] = `
 <input
+  aria-invalid={false}
```

Comment:
Isn't `{false}` in these (and following) snaps unexpected?

Reply:
We are defaulting this to `false` to match the "default" state (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute).

Without any validation this would still be false.  AXE didn't seem to find any issues with this.

Reply:
But shouldn't it be `aria-invalid="false"` rather than `{false}`? I haven't seen JSX syntax in snapshots before.

Reply:
nvmd. Bogus expectations on my part here. 😳